### PR TITLE
Remove `Runnable` interface.

### PR DESCRIPTION
**Pull request by a-hilaly**
This patch removes the `Runnable` interface and replaces it with
`Suite` struct.

There are two main changes:
- `Scenarios` or unique `Scenarios` can be represented in a suite that
  contains only that scenario.
- `gdt.From` now returns concrete object instead of a `Runnable`
  interface. Making it easier to predict the returned type - and
  advocates for modeling all tests as suites.

### CODE REVIEW

File: suite/from.go [hunk 1]
```diff
@@ -58,3 +63,16 @@ func FromDir(
...
 	return s, nil
 }
+
+// FromScenario encapsulates a given scenario in a fresh suite and returns it.
+func FromScenario(s *scenario.Scenario) *Suite {
+	return &Suite{
+		Path:        s.Path,
+		Name:        fmt.Sprintf("%s-%s", GeneratedSuiteNamePrefix, s.Name),
+		Description: s.Description,
+		// NOTE: require needs to be named to fixture?
```
Comment (jaypipes) [inline suite/from.go:73]: yeah... I need to rename this when we support suite fixtures...

**Review by jaypipes — CHANGES_REQUESTED** [review]
Great start on this, @a-hilaly! Thank you! :) A couple comments inline for you.

File: suite/from.go [hunk 2]
```diff
@@ -5,13 +5,18 @@
...
+	"fmt"
 	"os"
 	"path/filepath"
 
 	"github.com/gdt-dev/gdt/scenario"
 	"github.com/samber/lo"
 )
 
+const (
+	GeneratedSuiteNamePrefix = "generated-suite"
```
[the remark this answers is not in the document]
Comment (a-hilaly) [reply suite/from.go]: Changed to infer the name from the scenario path

**Review by jaypipes — APPROVED** [review]
awesome, thank you @a-hilaly! :)

**Pull request merged**